gateway/api/v1/views/get_jobs.py

```diff
@@ -0,0 +1,166 @@
+"""
+API V1: list jobs endpoint
+"""
+
+# pylint: disable=duplicate-code
+from typing import List, Optional
+
+from drf_yasg.utils import swagger_auto_schema
+from drf_yasg import openapi
+from rest_framework import status, permissions
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+from django.utils.dateparse import parse_datetime
+
+from django.conf import settings
+from api.models import Job
+from api.v1.views.utils import create_paginated_response, parse_positive_int
+from api.v1 import serializers as v1_serializers
+from api.v1.endpoint_decorator import endpoint
+from api.views.enums.type_filter import TypeFilter
+from api.use_cases.get_jobs import GetJobsUseCase
+
+
+def serialize_input(request):
+    """Parse and validate query parameters from the request."""
+    user = request.user
+
+    limit = parse_positive_int(
+        request.query_params.get("limit"), settings.REST_FRAMEWORK["PAGE_SIZE"]
+    )
+    offset = parse_positive_int(request.query_params.get("offset"), 0)
+
+    type_filter = None
+    type_param = request.query_params.get("filter")
+    if type_param:
+        try:
+            type_filter = TypeFilter(type_param)
+        except ValueError as exc:
+            raise ValueError(
+                f"Invalid type filter. Must be one of: {', '.join([e.value for e in TypeFilter])}"
+            ) from exc
+
+    status_filter = request.query_params.get("status")
+
+    created_after = None
+    created_after_param = request.query_params.get("created_after")
+    if created_after_param:
+        created_after = parse_datetime(created_after_param)
+        if created_after is None:
+            raise ValueError(
+                "Invalid created_after format. Use ISO 8601 format (e.g., '2024-01-01T00:00:00Z')"
+            )
+
+    function_name = request.query_params.get("function")
+
+    return {
+        "user": user,
+        "limit": limit,
+        "offset": offset,
+        "type_filter": type_filter,
+        "status": status_filter,
+        "created_after": created_after,
+        "function_name": function_name,
+    }
```

Comment:
I think this is a bit complex to read. Maybe a `Serializer` class fits better in here.